prime-router/metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml

```
@@ -54,4 +54,12 @@ elements:
     condition: '%resource.count() > 0'
     required: true
     schema: base/order-observation
-    resourceIndex: orderIndex
\ No newline at end of file
+    resourceIndex: orderIndex
+
+  - name: participation-request
+    resource: 'Bundle.entry.resource.ofType(Device)'
+    condition: 'Bundle.entry.resource.ofType(Device).udiCarrier.exists()'
+    constants:
+      hl7SegmentGroup: '/PATIENT_RESULT/PATIENT/PATIENT_OBSERVATION'
+    schema: ../common/participation-information
```

Nit: can we move this to match our naming schema?
I would expect:
```suggestion
    schema: ../resources/Device/PRT
```